layers/+lang/ipython-notebook/packages.el

```diff
@@ -9,214 +9,62 @@
-        "6" 'ein:notebook-worksheet-open-6th-km
-        "7" 'ein:notebook-worksheet-open-7th-km
-        "8" 'ein:notebook-worksheet-open-8th-km
-        "9" 'ein:notebook-worksheet-open-last-km
-        "+" 'ein:notebook-worksheet-insert-next-km
-        "-" 'ein:notebook-worksheet-delete-km
-        "x" 'ein:notebook-close-km
-        "u" 'ein:worksheet-change-cell-type-km
-        "fs" 'ein:notebook-save-notebook-command-km)
-
-      ;; keybindings for ipython notebook traceback mode
-      (spacemacs/set-leader-keys-for-major-mode 'ein:traceback-mode
-        "RET" 'ein:tb-jump-to-source-at-point-command
-        "n" 'ein:tb-next-item
-        "p" 'ein:tb-prev-item
-        "q" 'bury-buffer)
-
-      ;; keybindings mirror ipython web interface behavior
-      (evil-define-minor-mode-key 'insert 'ein:notebook-mode
-        (kbd "<C-return>") 'ein:worksheet-execute-cell-km
-        (kbd "<S-return>") 'ein:worksheet-execute-cell-and-goto-next-km)
-
-      ;; keybindings mirror ipython web interface behavior
-      (evil-define-minor-mode-key 'hybrid 'ein:notebook-mode
+    (mapc
+     (lambda (mode)
+       (evil-define-minor-mode-key
+        mode 'ein:notebook-mode
         (kbd "<C-return>") 'ein:worksheet-execute-cell-km
-        (kbd "<S-return>") 'ein:worksheet-execute-cell-and-goto-next-km)
-
-      (evil-define-minor-mode-key 'normal 'ein:notebook-mode
-        ;; keybindings mirror ipython web interface behavior
-        (kbd "<C-return>") 'ein:worksheet-execute-cell-km
-        (kbd "<S-return>") 'ein:worksheet-execute-cell-and-goto-next-km
-        "gj" 'ein:worksheet-goto-next-input-km
-        "gk" 'ein:worksheet-goto-prev-input-km)
-
-      ;; if this is not required then the following keygindings fail
-      (require 'ein-notebook)
-      (define-key ein:notebook-mode-map (kbd "M-j") 'ein:worksheet-move-cell-down-km)
-      (define-key ein:notebook-mode-map (kbd "M-k") 'ein:worksheet-move-cell-up-km)
-
-      (spacemacs|define-transient-state ipython-notebook
-        :title "iPython Notebook Transient State"
-        :doc "
- Operations on Cells^^^^^^            On Worksheets^^^^              Other
- ----------------------------^^^^^^   ------------------------^^^^   ----------------------------------^^^^
- [_k_/_j_]^^     select prev/next     [_h_/_l_]   select prev/next   [_t_]^^         toggle output
- [_K_/_J_]^^     move up/down         [_H_/_L_]   move left/right    [_C-l_/_C-S-l_] clear/clear all output
- [_C-k_/_C-j_]^^ merge above/below    [_1_.._9_]  open [1st..last]   [_C-o_]^^       open console
- [_O_/_o_]^^     insert above/below   [_+_/_-_]   create/delete      [_C-s_/_C-r_]   save/rename notebook
- [_y_/_p_/_d_]   copy/paste           ^^^^                           [_x_]^^         close notebook
- [_u_]^^^^       change type          ^^^^                           [_q_]^^         quit transient-state
- [_RET_]^^^^     execute"
-        :evil-leader-for-mode (ein:notebook-mode . ".")
```

Comment:
missed these, which causes users to lose the `,.` bindings for the transient mode and it's respective documentation minibuffer at the bottom.

Reply:
Indeed I missed `:evil-leader-for-mode`, but intentionally omitted the docstring.
I readded `:evil-leader-for-mode` and noticed the transient menu is unusable without the docstring.
So my conclusion is to eliminate the transient menu altogether since it cannot exist without a manually curated docstring, the manual upkeep of which I am opposed.

#13348